From 88353bf9931337efae5c06cad23306ff276d521e Mon Sep 17 00:00:00 2001
From: "Juuso \"Linda\" Lapinlampi" <linda@lindalap.fi>
Date: Mon, 8 Apr 2019 22:53:48 +0000
Subject: [PATCH] ch-opersys: Update referenced sections to FHS 3.0

The policy says in section 9.1.1 all files and directories must comply
with Filesystem Hierarchy Standard (FHS) 3.0. Later in section 9.1.2,
the references to FHS' section numbers were pointing to sections
apparently only sensible for an older FHS 2.3 document.

This fixes those references to the new numbers found in the FHS 3.0
document, and thus fixes the typos.

See: https://bugs.debian.org/922654
---
 policy/ch-opersys.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/ch-opersys.rst b/policy/ch-opersys.rst
index 59c92ec..6e0c020 100644
--- a/policy/ch-opersys.rst
+++ b/policy/ch-opersys.rst
@@ -127,8 +127,8 @@ empty.
 Note that this applies only to directories *below* ``/usr/local``, not
 *in* ``/usr/local``. Packages must not create sub-directories in the
 directory ``/usr/local`` itself, except those listed in FHS, section
-4.5. However, you may create directories below them as you wish. You
-must not remove any of the directories listed in 4.5, even if you
+4.9. However, you may create directories below them as you wish. You
+must not remove any of the directories listed in 4.9, even if you
 created them.
 
 If ``/etc/staff-group-for-usr-local`` does not exist, ``/usr/local``
-- 
2.20.1

